ynh-dev

```diff
@@ -306,6 +308,52 @@ elif [ "$1" = "use-git" ]; then
     done
 
 
+elif [ "$1" = "test" ]; then
+    check_yunohost_vm
+    VERSION=$2
+
+    for i in ${!packages[@]}; do
+        case ${packages[i]} in
+            yunohost)
+                # Pytest and tests dependencies
+                if ! type "pytest" > /dev/null; then
+                    echo "======================="
+                    echo "> Installing pytest ..."
+                    echo "======================="
+                    apt-get install python-pip
```

Comment:
Uh, actually the thing is that you're supposed to use this feature from inside the Yunohost instance (like use-git). But now that I mention it, it might be good to document it and check that yunohost is there (and installed ?) to avoid running this accidentally on the host system of the VM.